app/views/admin/organizations/index.html.haml

```diff
@@ -32,3 +32,4 @@
                   method: :get, class: 'btn btn-primary'
                 = link_to 'Delete', admin_organization_path(organization),
                   method: :delete, class: 'btn btn-danger', data: { confirm: "Warning: This will delete #{organization.name} and all its data which includes data for all conferences within #{organization.name}. Do you really want to continue?" }
+                = link_to 'Add Conference', new_admin_conference_path, method: :get, class: 'btn btn-info'
```

Comment:
I'd also pass the organization, so that it is pre-selected in the new conference form.

Reply:
@Buraisx would you like to add this as well before merging the PR?